apps/experiments/versioning.py

```diff
@@ -78,8 +89,16 @@ def get_field(self, field_name: str) -> VersionField:
 
     @property
     def fields_grouped(self):
-        # TODO: Return fields with group info for display purposes
-        pass
+        groups = defaultdict(dict)
+        for field in self.fields:
+            group_name = field.group_name
+            group_info = groups.get(group_name, FieldGroup(name=group_name))
+            group_info.show = group_info.show or bool(field.raw_value) or bool(field.changed)
+            group_info.has_changed_fields = group_info.has_changed_fields or field.changed
+            # We need to always include the field
+            group_info.fields.append(field)
+            groups[group_name] = group_info
+        return groups.items()
```

Comment:
or just `list(groups.values())` since the name is already in the group object

Note: wrapping with list to avoid returning a dict view object.